csharp/native/HelloWorld/Client/Program.cs

```diff
@@ -0,0 +1,178 @@
+                                                               DefaultTaskOption = new TaskOptions
+                                                                                   {
+                                                                                     MaxDuration = Duration.FromTimeSpan(TimeSpan.FromHours(1)),
+                                                                                     MaxRetries  = 2,
+                                                                                     Priority    = 1,
+                                                                                     PartitionId = partition,
+                                                                                   },
+                                                               PartitionIds =
+                                                               {
+                                                                 partition,
+                                                               },
+                                                             });
+
+      // Create the result id
+      var resultId = Guid.NewGuid()
+                         .ToString();
+
+      var createTaskReply = await submitterClient.CreateTasksAsync(createSessionReply.SessionId,
+                                                                   null,
+                                                                   new[]
+                                                                   {
+                                                                     // Task request with payload for the task
+                                                                     // Also contains the list of results that will be created by the task
+                                                                     new TaskRequest
+                                                                     {
+                                                                       ExpectedOutputKeys =
+                                                                       {
+                                                                         resultId,
+                                                                       },
+                                                                       // Avoid unnecessary copy but data could be modified during sending if the
+                                                                       // reference to the object is still available
+                                                                       // This is not the case here
+                                                                       Payload = UnsafeByteOperations.UnsafeWrap(Encoding.ASCII.GetBytes("Hello")),
+                                                                     },
+                                                                   })
+                                                 .ConfigureAwait(false);
```

Comment:
We should probably remove this rule for samples. It only adds noise.

Reply:
That's not the only rule we have to change.
But in another pr